src/app/matches/matches-timeline.tsx

```
@@ -0,0 +1,119 @@
+        const date = new Date(timeStr);
+        if (!Number.isNaN(date.getTime())) return date.getHours() * 60 + date.getMinutes();
+        return 0;
+    };
+
+    let minMin = 24 * 60;
+    let maxMin = 0;
+
+    const matchesWithTime = matches.map((match) => {
+        const start = getMinutes(match.startTime);
+        const end = getMinutes(match.endTime) || start + 30; // default 30m if no end time
+        if (start < minMin && start > 0) minMin = start;
```

ESLint

[error] Error: Cannot reassign variable after render completes

Reassigning `minMin` after render has completed can cause inconsistent behavior on subsequent renders. Consider using state instead.

/home/jailuser/git/src/app/matches/matches-timeline.tsx:30:42
  28 |         const start = getMinutes(match.startTime);
  29 |         const end = getMinutes(match.endTime) || start + 30; // default 30m if no end time
> 30 |         if (start < minMin && start > 0) minMin = start;
     |                                          ^^^^^^ Cannot reassign `minMin` after render completes
  31 |         if (end > maxMin) maxMin = end;
  32 |         
  33 |         const tableHref = match.link("competitionTable")?.href;

(react-hooks/immutability)